axiom/tests/test_cmd_txndata.c

```diff
@@ -55,10 +55,6 @@ int nr_get_daemon_fd(void) {
   return 0;
 }
 
-nrapp_t* nr_app_verify_id(nrapplist_t* applist NRUNUSED,
```

Comment:
Why was this removed?

Reply:
Please see https://github.com/newrelic/newrelic-php-agent/commit/fbd8b2ff08e7e8ce26ff63afbc66c2871467cb7b commit message for an explanation why this is required to be defined here too.